cluster/log.go

```diff
@@ -16,19 +16,18 @@ import (
 	"fmt"
 
 	"github.com/hashicorp/raft"
-	raftboltdb "github.com/hashicorp/raft-boltdb/v2"
 )
 
-type rLog struct {
-	*raftboltdb.BoltStore
-}
+func (st *Store) LastAppliedCommand() (uint64, error) {
+	if st.logStore == nil {
+		return 0, nil
```

Comment:
nit: Would it make sense to return an error if we are trying to get the last applie command where logStore doesn't exists ?

Reply:
yeah i was going back and forth on it. when the log store is nil, it means `LastAppliedCommand()`  is called before `init()`. maybe to be in safe the zone will return an error then